Caboodle/Sms/Sms.android.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Android.App;
+using Android.Content;
+using Android.Net;
+using Android.Telephony;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Sms
+    {
+        static bool GetCanSendSms() => true;
```

Comment:
Are we sure that every device can send an Sms? this seems like a false assumption. We should validate that there is an app installed that can handle the "smsto:" we should probably make a Platform internal static to validate Intents and the Uri.